.circleci/config.yml

```diff
@@ -179,6 +190,14 @@ workflows:
       - sonar:
           requires:
             - test
+      - dependency-track:
+          filters:
+            branches:
+              only: master
+          requires:
+            - sonar
```

Comment:
does this really depend on the sonar job?
Looks more like it depends on the  Build Job

Reply:
Technically, you are right. It could depend on the build job. However, since this is publishing to the Dependency Track server, we think it is better if we do it when everything else is good including Sonar.

Reply:
Fair enough... however, as a matter to point out, since tmc-auth is Open Source, the sequential nature isn't significant at all, and having it dependent on the build step would open it up to parallel execution.